.travis.yml

```diff
@@ -15,4 +15,4 @@ cache:
 
 script:
   - 'if [ "${TRAVIS_EVENT_TYPE}" != "cron" ]; then docker run --rm -ti -v "${HOME}"/.portage:/usr/portage -v "${PWD}":/overlay -w /overlay gentoo/stage3-amd64:latest /overlay/tests/runtests.sh; fi'
-  # - 'if [ "${TRAVIS_EVENT_TYPE}" == "cron" ]; then docker run --rm -ti -v "${PWD}/tests/newversionchecker.toml":/app/newversionchecker.toml -e GITHUB_API_TOKEN simonvanderveldt/newversionchecker; fi'
+  - 'if [ "${TRAVIS_EVENT_TYPE}" == "cron" ]; then docker run --rm -ti -v "${PWD}/tests/newversionchecker.toml":/app/newversionchecker.toml -e GITHUB_API_TOKEN simonvanderveldt/newversionchecker; fi'
```

Comment:
this belongs in a separated commit, It has nothing to do with media-sound/drumkv1